### ISSUE TITLE
Synchronisation with DropSync

### ISSUE THREAD

**Issue by Villard** [eleybourn/Book-Catalogue#745]
Hello
I'm a great fan of BookCatalogue !

Until now I synchronized only the backup  I make manually every two weeks. But my phone crashed and I've lost the last 15 days. So I'm now setting the Dropsync to sync and save the Bookcatalogue folder of my android with my Dropbox, and I hope that every change made in the Database will be synchronized.
I just want to check the Bookcatalogue database itself is effectively stored in this folder and will be synchronized. 
I ask this question because  I see  in the bookcatalogue folder, all my  2200 jpeg covers and one file called "covers.db" which is only 30 Kb. I am not sure this "covers.db" is the database of the app. 
Thanks for you reply
Martin

**Comment by Grunthos** [comment]
No; the database is stored internally on the phone. You will still need to 
"Backup to Archive" periodically.

**Comment by Villard** [comment]
Hello
Thanks for your reply. So I still need not to forget to backup.
Martin
